data/user-groups.yml

```
@@ -1250,3 +1250,9 @@
     position:
       lat: 9.2034963
       lng: 12.4953899
+ - name: Morocco KUG
```

Thanks for the contribution!
Slight inaccuracy in the format:
```suggestion
  - name: Morocco KUG
```